recipes/mne-python_1.0/construct.yaml

```diff
@@ -88,6 +88,12 @@ specs:
   - neurodsp
   - bycycle
   - fooof
+  # MNE-BIDS-Pipeline
+  - coloredlogs
+  - json_tricks
+  - fire
+  # OpenNeuro.org data access
+  - openneuro-py
```

Comment:
Yes it could get annoying at one point, but then again we're not neither changing the Pipeline deps super frequently, nor are we going to release new installers every few days or so. So I suppose the workload will be pretty small.

As for automating this, unfortunately I don't think there's a trivial way to do it: we cannot simply feed the Pipelines `requirements.txt` to the `constructor` that builds the installer, if that's what you had in mind